manifests/params.pp

```diff
@@ -159,6 +159,7 @@
   $server_database_schema                   = undef
   $server_database_socket                   = undef
   $server_database_user                     = 'zabbix_server'
+  $server_database_tablespace               = 'pg_default'
```

Comment:
Can we default this to `undef`? That's the default of the `tablespace` parameter of `postgresql::server::db`:

https://github.com/puppetlabs/puppetlabs-postgresql/blob/337595646688eec3c4f26ace51d1b3e8b7f5ad07/manifests/server/db.pp#L22

Reply:
Done!